src/components/common/CustomInput/CustomInput.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+interface Props {
+  id: string;
+  type: string;
+  value?: string | number;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  placeholder?: string;
+  className?: string;
+}
+
+const CustomInput = ({ id, type, value, placeholder, onChange, className }: Props) => {
+  return (
+    <input
+      id={id}
+      type={type}
+      value={value}
+      className={className}
+      onChange={onChange}
+      placeholder={placeholder}
+    />
```

Comment:
```suggestion
    <input
      id={id}
      type={type}
      value={value}
      className={className}
      onChange={onChange}
      placeholder={placeholder}
      {...props}
    />
```
다른 스타일 넣어도 적용되도록 하는것도 좋을것 같습니다. 다만, 그럴경우 타입을 object로 하면 안되어서 다시 또 타입을 정의해야하는데 그런경우 어떤 타입으로 정의해야할지 어렵네요!

Reply:
1) Label로 하면 기본 tag인 `label`과 네이밍이 완전하게 같아서 혼동을 줄 수 있을 것 같은데 어떻게 생각하시나요?
2) ...props는 추가적인 요소가 있을 때 마다 추가해주면 어떨까요?